ui/components/App.vue

```diff
@@ -13,22 +36,72 @@
 </template>
 
 <script>
-import { mapGetters, mapActions } from 'vuex'
+import { mapGetters, mapActions, mapState } from 'vuex'
 import AccountsList from './accounts/AccountsList.vue'
 export default {
   name: 'App',
   components: { AccountsList },
   computed: {
     ...mapGetters('Accounts', ['isInitialized', 'getAccountsSorted']),
+    ...mapState('Accounts', ['selectedAccounts']),
+
     accounts () {
       return this.getAccountsSorted
+    },
+
+    selectedAccountsAmount () {
+      return this.selectedAccounts.length
+    },
+
+    selectionInfoText () {
+      const translated = this.$ngettext('%{ amount } selected user', '%{ amount } selected users', this.selectedAccountsAmount)
+
+      return this.$gettextInterpolate(translated, { amount: this.selectedAccountsAmount })
+    },
+
+    actions () {
+      const actions = []
+      const isAnyAccountDisabled = this.selectedAccounts.some(account => !account.accountEnabled)
```

Comment:
If you'd count the number of disabled accounts you would iterate only once and then compare numbers (`number of disabled accounts > 0` and `number of disabled accounts < total number of accounts`).